Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM golang:1.14.1
-EXPOSE 8080
+FROM golang:1.14.1 as builder
 WORKDIR /go/src/vvgo
 COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o vvgo
 
-RUN go install -v ./...
-
-CMD ["vvgo"]
\ No newline at end of file
+FROM alpine:latest
+RUN apk --no-cache add ca-certificates
```

Comment:
Why not `gcr.io/distroless/static`? If we're going alpine, we may as well go distroless.

Reply:
We should be able to do scratch image, except that go isn't _truly_ a self contained binary. lmao